cli/sql.go

```diff
@@ -49,22 +56,52 @@ func runTerm(cmd *cobra.Command, args []string) {
 		_ = liner.Close()
 	}()
 
+	fmt.Fprint(osStdout, infoMessage)
```

Comment:
We should guard this (and the prompt) with a check for `isatty()` (wherever that is exposed in go), so you can also run `cockroach sql < somefile.sql`.